sdkproject/Assets/Mapbox/Unity/DataContainers/MapboxEnums.cs

```diff
@@ -143,14 +143,16 @@ public enum ExtrusionType
 		None,
 		[Description("Extrude features using the property value.")]
 		PropertyHeight,
-		[Description("Extrude features using the property value. Values lower than min value are clamped at min value.")]
+		[Description("Extrude features using the property value. Extrusion from features minimum height value. Results in flat tops.")]
```

Comment:
"Extrusion from features minimum height value" -> can we phrase this differently? Is it fair to say "Sets the minimum height of extrusions"? Same comment for max height ("Sets the maximum...")

Reply:
Actually this doesn't set extrusions minimum value. Choses the vertex with minimum height and extrudes the height from there for all features. 
For example, if you had a building on a hill, the footprint will have different heights at different corners, when choosing MinHeight extrusion, the lowest vertex will be extruded by the "height" value and the rest will be extruded to a value which results in flat tops.

Reply:
Ahh got it. Now I better understand your definition. How about:

"Sets height based on property's minimum height, if height isn't uniform"
"Sets height based on feature's minimum height"
"Choses property's minimum height and sets as uniform height"
"Choses property's minimum height and sets that as base for extruding all features"

Pick whatever you think works best!